### ISSUE TITLE
BTCV

### ISSUE THREAD

**Issue by Adoreeeeee** [ljwztc/CLIP-Driven-Universal-Model#45]
Again bothering you. About BTCV paper results reproduction. What I haven't figured out is that only the training set and the validation set are used in the five-fold cross-validation, so what does the test set do? Is the test set part of the training set when the entire model is pre-trained? Looking forward to your response.

**Comment by ljwztc** [comment]
There is five fold in [file](https://github.com/ljwztc/CLIP-Driven-Universal-Model/blob/main/dataset/BTCV_folds.json). We conduct five-cross validation according to this file. More details please refer to #26
